### ISSUE TITLE
Add task skill UI to the new task page

### ISSUE THREAD

**Issue by begedin** [code-corps/code-corps-ember#1046]
# Problem

Ideally, we want to be able to assign skills to a task while creating the task, on the new task page or within the new task form.

This is trickier then assigning/unassigning to an existing task, but should be doable

## Proposed solution

- [ ] new task form component should also hold a skills-typeahead, as well as a skill list
- there is also a list of project skills, which we can use to add skills listed on the project quickly with a single click - this can be done separately in #1052 
- [ ] instead of the classical approach of task-skills being created and destroyed when using this UI, we instead store/remove them to/from an array held by the component
- [ ] on hitting the form submit button, the save action also passed out a list of skills, which are then created as task skills alongside the task itself, after the task was created

## Optional improvement

- Once the implementation above is done, we can look into optimizing it (probably initially not necessary) by modifying our task create request to also embed task skills. This would require API changes as well.

## UI

Sidebar of the new task page:

![image](https://cloud.githubusercontent.com/assets/1925540/22921399/c292ecd0-f299-11e6-9d21-8b60c2a88cba.png)

## References

Progress on: #1042

**Comment by JoshSmith** [comment]
@begedin is this good to close now?

**Comment by begedin** [comment]
This part can be closed. The part in #1052 is not done yet.
